core/store/models/key.go

```diff
@@ -17,8 +17,8 @@ import (
 //
 // By default, a key is assumed to represent an ethereum account.
 type Key struct {
-	ID        int32
```

Comment:
The PR desc doesn't say where this bug is creating issues?!
I susspect it's because we're using the key.ID to grab an advisory lock in the broadcaster!